hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/event/EventType.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.alerts.api.model.event;
+
+/**
+ * What the trigger produces.
+ *
+ * @author jay shaughnessy
+ * @author lucas ponce
+ */
+public enum EventType {
+    ALERT, EVENT
```

Comment:
Yesterday I was still thinking on the Heritage vs Composition approaches to represent the relationship between Alert and Event.
From different angles I see both approaches correct which is hard because we need to choose one and it's unclear which is better.
Here some of my arguments we can talk later:
- From natural language I see that a event/happening might need an Alert. But there are nuances that make me think that Event is the object and the Alert is the subject who operates with, so in that sense the composition pattern maps better.
- Also, to update an Alert shouldn't mean that an Event is being modified (as a happening has a time).
- As per requeriments, we might introduce new things on Alert (as comments, history of transitions, and others), I see that heritage is perhaps to overload the Event concept.
Anyway, I think this is better to comment in a talk as at the end we are talking about details/nuances.